gradle.properties

```diff
@@ -1,4 +1,4 @@
-kotlinVersion=1.1.3
+kotlinVersion=1.2.0-beta-88
```

Comment:
Let’s just agree here that this will not be released until stable Kotlin release.

You have no idea how much pain we have with JUnit 5 because Android Studio does not support it properly... :-(

Reply:
The problem is that JB doesn't announce planned dates for releases, so this change can potentially keep us from 2.x release for some time…

Reply:
On the positive side, it shouldn't keep us from releasing `rc/preview` builds!